parsl/configs/cooley_htex_multinode.py

```diff
@@ -0,0 +1,31 @@
+from parsl.config import Config
+from parsl.executors import HighThroughputExecutor
+from parsl.addresses import address_by_hostname
+from parsl.launchers import MpiRunLauncher
+from parsl.providers import CobaltProvider
+
+
+config = Config(
+    executors=[
+        HighThroughputExecutor(
+            label="cooley_htex",
+            worker_debug=False,
+            cores_per_worker=1,
+            address=address_by_hostname(),
+            provider=CobaltProvider(
+                queue='debug',
+                account='',  # project name to submit the job
```

Comment:
Can the config run without this? If not, let's rename `YOUR_ACCOUNT`

Reply:
It does not work. Fixed now